### DISCUSSION TITLE
[source-appstore] Add `ratings` and `reviews` streams

### DISCUSSION THREAD

**Discussion by d1bevz** [airbytehq/airbyte#34564] [Connector Ideas and Features]
### Connector Name

Appstore

### Connector Version

0.2.6

### What step the error happened?

Configuring a new connector

### Relevant information

In docs you say that many types of services could be added to the connector, I would appreciate it if you could add [ratings and reviews stream](https://developer.apple.com/help/app-store-connect/monitor-ratings-and-reviews/view-ratings-and-reviews)

### Contribute

- [x] Yes, I want to contribute

**Comment by marcosmarxm** [comment]
Move this to Discussions

**Reply by d1bevz** [reply]
I made an issue as it was said in [docs](https://docs.airbyte.com/integrations/sources/appstore)

[a comment by an automated account is not shown]
